packages/config/config/devices/0x0086/zw141.json

```diff
@@ -0,0 +1,229 @@
+		{
+			"productType": "0x0303",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3132
+		},
+		{
+			"productType": "0x0903",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3269
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"35": {
+			"label": "Moving Time From Up (left) to Down (right) For Curtain",
+			"description": "The available range for time set is 1 to 127 seconds",
+			"valueSize": 1,
+			"unit": "seconds",
+			"minValue": 1,
+			"maxValue": 127,
+			"defaultValue": 127,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"80": {
+			"label": "Notification To Send to The Associated Devices (Group 1) on State Change",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Send nothing",
+					"value": 0
+				},
+				{
+					"label": "Send basic cc report",
+					"value": 1
+				}
+			]
+		},
+		"85": {
+			"label": "Operation Mode of External Switch",
+			"description": "Refer to the user manual for the description of operation mode 1 or 2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Operation mode 1",
+					"value": 0
+				},
+				{
+					"label": "Operation mode 2",
+					"value": 1
+				}
+			]
+		},
+		"120": {
+			"label": "External Switch Mode For S1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unidentified mode",
+					"value": 0
+				},
+				{
+					"label": "Reserved",
+					"value": 1
+				},
+				{
+					"label": "3-way switch mode",
+					"value": 2
+				},
+				{
+					"label": "Momentary push button mode",
+					"value": 3
+				},
+				{
+					"label": "Enter automatic identification mode",
+					"value": 4
+				}
+			]
+		},
+		"121": {
+			"label": "External Switch Mode For S2",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 4,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Unidentified mode",
+					"value": 0
+				},
+				{
+					"label": "Reserved",
+					"value": 1
+				},
+				{
+					"label": "3-way switch mode",
+					"value": 2
+				},
+				{
+					"label": "Momentary push button mode",
+					"value": 3
+				},
+				{
+					"label": "Enter automatic identification mode",
+					"value": 4
+				}
+			]
+		},
+		"248": {
+			"label": "S1/S2 Functionality",
+			"valueSize": 1,
+			"minValue": -128,
+			"maxValue": 4,
+			"defaultValue": 83,
+			"unsigned": true,
```

Comment:
These value ranges and choices don't make senses. Could you check the manual? The default, 83, isn't an option. And the options are odd to me. Is this a partial parameter situation where you can set more than one at once?

Reply:
I'm actually not sure on this. I had gotten that info [from here](http://manuals-backend.z-wave.info/make.php?lang=en&sku=AEOEZW141&cert=), but am now seeing the following in a manual. Any thoughts?

<img width="768" alt="Screen Shot 2021-02-09 at 9 04 20 PM" src="https://user-images.githubusercontent.com/1899920/107454220-977ff800-6b1a-11eb-9eab-20816b274a20.png">

Reply:
Ah, this is a bitmask that slipped through. I'll take a look at this tomorrow but we need to break it apart.